src/Neo.CLI/config.fs.testnet.json

```diff
@@ -36,6 +36,12 @@
     "MaxTraceableBlocks": 2102400,
     "InitialGasDistribution": 5200000000000000,
     "ValidatorsCount": 7,
+    "Hardforks": {
+      "HF_Aspidochelone": 210000,
+      "HF_Basilisk": 2680000,
+      "HF_Cockatrice": 3967000,
+      "HF_Domovoi": 4144000
+    },
```

Comment:
For NeoFS testnet this section is not needed, we have all hardforks enabled starting from the genesis. Just remove it.

Reply:
Removed